exercises/bank-account/.meta/version

```diff
@@ -1 +1 @@
-0.0.0
+1.0.0
```

Comment:
```suggestion
0.0.0
```

For tests that don't have data, we are going to leave it at `0.0.0` like the JavaScript and TypeScript group. I'll clarify in the issue.

Reply:
Sure, thanks for pointing out.